packages/webcert/src/feature/certificate/Inputs/UeTypeahead.tsx

```diff
@@ -68,9 +68,14 @@ const UeTypeahead: React.FC<Props> = ({ question, disabled }) => {
       if (newText.length === 0) {
         setOpen(false)
       }
-      setSuggestions(
-        questionConfig.typeAhead.filter((suggestion: string) => suggestion.toLowerCase().indexOf(newText.toLowerCase()) >= 0).sort()
-      )
+
+      const sortResult = [
+        ...questionConfig.typeAhead.filter((suggestion) => suggestion.toLowerCase().startsWith(newText.toLowerCase())),
+        ...questionConfig.typeAhead.filter((suggestion) => suggestion.toLowerCase().indexOf(newText.toLowerCase()) >= 0).sort(),
+      ]
+      const newResult = new Set(sortResult)
```

Comment:
Instead of using Set and double filters to create a list of unique values, we could perhaps do one run of filtering.

```typescript
questionConfig.typeAhead.filter((suggestion) => {
  const haystack = suggestion.toLowerCase()
  const needle = newText.toLowerCase() 
  return haystack.startsWith(needle)  && haystack.indexOf(needle) >= 0
}))
```

Reply:
It is necessary when you, for example, search for 'Ors', then you expect to get 'Orsa' at the top and also other words that contain 'Ors' . If we only use IndexOf we get this result (also added a sort()):
![image](https://user-images.githubusercontent.com/79509664/201044759-9f2f57de-bad5-4278-bcc4-fed175ba8e37.png)

Reply:
good idéa!